### PR TITLE
Add uploader for integration test

### PR DESCRIPTION
**Pull request by ping-ke**
Add uploader for integration test: randomly generate bytes and upload to newly created contracts integration test.

### CODE REVIEW

File: ituploader.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const {ethers, Contract} = require("ethers");
+const crypto = require('crypto');
+const {EthStorage} = require("./dist/index.cjs.js");
+const {NodeFile} = require("./dist/file.cjs.js");
+const fs = require('fs');
```
Comment (iteyelmp) [inline ituploader.js:5]: This doesn't seem to be related to the SDK. Should we create a separate repo for testing?

Comment (qzhodl) [reply ituploader.js]: The the same question here, what is the motivation of adding this script to the SDK?
